Makefile

```diff
@@ -195,6 +205,11 @@ node_modules/angular/angular.min.js node_modules/angular-animate/angular-animate
 	mkdir -p $(dir $@)
 	touch $@
 
+.build/bower_components.timestamp: bower.json
+	bower install
```

Comment:
This did not work in my case. I think you need to use `./node_modules/bower/bin/bower`. Maybe it works for you because you already have bower globally installed.